lib/ansible/modules/cloud/vmware/vmware_guest_facts.py

```diff
@@ -63,11 +63,12 @@
 '''
 
 EXAMPLES = '''
-- name: Gather VM facts
+- name: Gather facts from standalone ESXi server having datacenter as 'ha-datacenter'
   vmware_guest_facts:
     hostname: 192.168.1.209
     username: administrator@vsphere.local
```

Comment:
If this is a standalone hypervisor, shouldn't the username be `root`? administrator@vsphere.local is not available on a standalone hypervisor